package.json

```diff
@@ -35,45 +35,45 @@
   },
   "devDependencies": {
     "@gravitational/build": "workspace:*",
-    "@storybook/addon-toolbars": "^8.3.4",
-    "@storybook/components": "^8.3.4",
-    "@storybook/react": "^8.3.4",
-    "@storybook/react-vite": "^8.3.4",
+    "@storybook/addon-toolbars": "^8.4.0",
+    "@storybook/components": "^8.4.0",
+    "@storybook/react": "^8.4.0",
+    "@storybook/react-vite": "^8.4.0",
     "@storybook/test-runner": "^0.19.1",
-    "@testing-library/jest-dom": "^6.5.0",
+    "@testing-library/jest-dom": "^6.6.3",
```

Comment:
@bl-nero Do you mind looking at [the failing `toHaveStyle` tests](https://github.com/gravitational/teleport/actions/runs/11664459496/job/32475027347?pr=48268#step:7:700)? I'm not sure what might have caused them, but from the list of updates packages, `@testing-library/jest-dom` is the most suspicious.

Reply:
The issue is that suddenly, elements started being rendered in an active+hover state by default. I don't know exactly what is responsible just yet, but there are related issues on [testing-library](https://github.com/testing-library/react-testing-library/issues/1225) and [jsdom](https://github.com/jsdom/jsdom/issues/3607). I don't yet know how to solve it, but I'm working on it.